### PR TITLE
Updates to documentation for initial setup

### PR DESCRIPTION
**Pull request by aglyle**
Adds documentation to bypass setup errors.

### CODE REVIEW

File: treeshop.md [hunk 1]
```diff
@@ -26,15 +26,20 @@ Clone this repository:
...
     mkdir -p treeshop/primary/original/TEST treeshop/downstream
 
-Copy the TEST fastq samples into the storage hierarchy
-  
+Copy the TEST fastq samples into the storage hierarchy:
+
     cp samples/*.fastq.gz treeshop/primary/original/TEST/
 
-Spin up a single cluster machine:
+Spin up a single cluster machine (make sure you have created your SSH key):
```
Comment (e-t-k) [inline treeshop.md]: Provide instructions for:
a) checking if you already have an "~/.ssh/id_rsa" private+public keypair files that don't require a password
b) if you don't have a keypair, how to create it

Comment (e-t-k) [reply treeshop.md]: Another thing to incorporate at the end: 
- how to spin down machines when done
 - the info from Rob's April 12 email "Floating IPs" about how to clear the IP addresses after spinning down machines.

**Comment by e-t-k** [comment]
Thank you @aglyle ; I'm satisfied.

@klearned , passing this to you -- could you please do a readthrough of the new treeshop.md for potential improvements?

**Comment by klearned** [comment]
Yes, I'll read through the new treeshop.md today

**Comment by aglyle** [comment]
Thanks for the comments @klearned!
@e-t-k can you take a final look at these changes and I'll ping Rob to merge the branch.

**Comment by e-t-k** [comment]
@aglyle looks great. I approve this doc.

**Comment by aglyle** [comment]
@rcurrie Updates to the treeshop.md ready to be merged with master.  Let me know if there's anything you want me to change.

**Comment by rcurrie** [comment]
Good work - thank you for filling in pieces. I made a bunch of comments (feel free to push back!). In general I want to be careful about helping someone use this vs. helping someone that shouldn’t be using this.

**Comment by aglyle** [comment]
@rcurrie Incorporated your comments.  Removed sections that we expect users to know (SSH Key creation), unsure if you wanted to keep PATH setup or not, but it's removed in the latest version.  Also moved documentation for shutting down select machines and different fabfiles to links in the options.  Removed shutting down IPs as it is Treehouse specific.

**Pull request merged**